### PR TITLE
Chore: Don't install DevPackages in docker image 

### PR DESCRIPTION
**Pull request by McTonderski**
To remove possible security concern points I want to remove installing devPackages which are not needed in docker image.

Any suggestion is highly anticipated

### CODE REVIEW

**Review by juanicabanas — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
+FROM base AS dev-dependencies
 
+COPY package.json yarn.lock ./
 RUN yarn install --pure-lockfile
+
+# Build stage
+FROM dev-dependencies AS build
+
+COPY . ./
 RUN yarn run build
```
Comment (AgnesToulet) [inline Dockerfile:23]:
```suggestion
# Build stage
FROM base AS build

COPY . ./

RUN yarn install --pure-lockfile
RUN yarn run build
```

Comment (AgnesToulet) [reply Dockerfile:23]: @McTonderski Could you take a look at this suggestion, please? It simplifies a bit the code while still keeping the dev dependencies out of the final image

**Comment by AgnesToulet** [comment]
Based on this PR, I opened and merged this one that contains a few updates: https://github.com/grafana/grafana-image-renderer/pull/575

Thanks for starting the work on this!

File: Dockerfile (hunk 1 above)
Comment (McTonderski) [reply Dockerfile:23]: hey, Thank you for pushing that MR !

**Pull request closed without merging**